Formula/p/protobuf.rb

```diff
@@ -22,26 +22,32 @@ class Protobuf < Formula
 
   depends_on "cmake" => :build
   depends_on "abseil"
-  depends_on "jsoncpp"
-
   uses_from_macos "zlib"
 
+  on_macos do
+    # We currently only run tests on macOS.
+    # Running them on Linux requires rebuilding googletest with `-fPIC`.
+    depends_on "googletest" => :build
```

Comment:
Is there any reason not to do this?

Reply:
Wasn't sure if it's worth rebuilding the bottles for, but we could do it. We'd need to rebuild abseil too though:

https://github.com/Homebrew/homebrew-core/blob/14494116bec4249ed96a4fdb9cac82d9e193d06a/Formula/a/abseil.rb#L30-L34

Reply:
Let's do this in a follow-up PR if desired.